pypeit/spectrographs/bok_bc.py

```diff
@@ -115,12 +115,12 @@ def raw_header_cards(self):
         downstream output files for configuration identification.
 
         The list of raw data FITS keywords should be those used to populate
-        the :meth:`~pypeit.spectrograph.Spectrograph.configuration_keys`
-        or are used in :meth:`~pypeit.spectrograph.Spectrograph.config_specific_par`
+        the :meth:`~pypeit.spectrographs.spectrograph.Spectrograph.configuration_keys`
+        or are used in :meth:`~pypeit.spectrographs.spectrograph.Spectrograph.config_specific_par`
```

Comment:
They aren't different in how it shows up in HTML.  Someone else fussed at me about using `meth` over `func` in the fluxing documentation.  Since these are class methods instead of stand-alone functions, I chose `meth`.

See, for example, https://stackoverflow.com/questions/34848253/whats-the-difference-in-behaviour-between-func-and-meth-roles-in-python-sph